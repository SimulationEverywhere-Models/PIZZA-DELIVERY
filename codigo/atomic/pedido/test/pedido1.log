Mensaje I / 00:00:00:000 / Root(00) para top(01)
Mensaje I / 00:00:00:000 / top(01) para pedido(02)
Mensaje D / 00:00:00:000 / pedido(02) / ... para top(01)
Mensaje D / 00:00:00:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:01:000 / Root(00) / pedido_in /      1.00000 para top(01)
Mensaje X / 00:00:01:000 / top(01) / pedido_in /      1.00000 para pedido(02)
Mensaje D / 00:00:01:000 / pedido(02) / 00:00:00:030 para top(01)
Mensaje D / 00:00:01:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:01:030 / Root(00) para top(01)
Mensaje * / 00:00:01:030 / top(01) para pedido(02)
Mensaje Y / 00:00:01:030 / pedido(02) / pedido_out /      1.00000 para top(01)
Mensaje D / 00:00:01:030 / pedido(02) / ... para top(01)
Mensaje Y / 00:00:01:030 / top(01) / pedido_out /      1.00000 para Root(00)
Mensaje D / 00:00:01:030 / top(01) / ... para Root(00)
Mensaje X / 00:00:02:000 / Root(00) / pedido_in /      2.00000 para top(01)
Mensaje X / 00:00:02:000 / top(01) / pedido_in /      2.00000 para pedido(02)
Mensaje D / 00:00:02:000 / pedido(02) / 00:00:00:030 para top(01)
Mensaje D / 00:00:02:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:02:030 / Root(00) para top(01)
Mensaje * / 00:00:02:030 / top(01) para pedido(02)
Mensaje Y / 00:00:02:030 / pedido(02) / pedido_out /      2.00000 para top(01)
Mensaje D / 00:00:02:030 / pedido(02) / ... para top(01)
Mensaje Y / 00:00:02:030 / top(01) / pedido_out /      2.00000 para Root(00)
Mensaje D / 00:00:02:030 / top(01) / ... para Root(00)
Mensaje X / 00:00:03:000 / Root(00) / esperar_p /      1.00000 para top(01)
Mensaje X / 00:00:03:000 / top(01) / esperar_p /      1.00000 para pedido(02)
Mensaje D / 00:00:03:000 / pedido(02) / ... para top(01)
Mensaje D / 00:00:03:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:04:000 / Root(00) / pedido_in /      3.00000 para top(01)
Mensaje X / 00:00:04:000 / top(01) / pedido_in /      3.00000 para pedido(02)
Mensaje D / 00:00:04:000 / pedido(02) / ... para top(01)
Mensaje D / 00:00:04:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:05:000 / Root(00) / pedido_in /      4.00000 para top(01)
Mensaje X / 00:00:05:000 / top(01) / pedido_in /      4.00000 para pedido(02)
Mensaje D / 00:00:05:000 / pedido(02) / ... para top(01)
Mensaje D / 00:00:05:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:06:000 / Root(00) / pedido_in /      5.00000 para top(01)
Mensaje X / 00:00:06:000 / top(01) / pedido_in /      5.00000 para pedido(02)
Mensaje D / 00:00:06:000 / pedido(02) / ... para top(01)
Mensaje D / 00:00:06:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:07:000 / Root(00) / listo_p /      1.00000 para top(01)
Mensaje X / 00:00:07:000 / top(01) / listo_p /      1.00000 para pedido(02)
Mensaje D / 00:00:07:000 / pedido(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:07:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:07:000 / Root(00) para top(01)
Mensaje * / 00:00:07:000 / top(01) para pedido(02)
Mensaje Y / 00:00:07:000 / pedido(02) / pedido_out /      3.00000 para top(01)
Mensaje D / 00:00:07:000 / pedido(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:07:000 / top(01) / pedido_out /      3.00000 para Root(00)
Mensaje D / 00:00:07:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:07:030 / Root(00) para top(01)
Mensaje * / 00:00:07:030 / top(01) para pedido(02)
Mensaje Y / 00:00:07:030 / pedido(02) / pedido_out /      4.00000 para top(01)
Mensaje D / 00:00:07:030 / pedido(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:07:030 / top(01) / pedido_out /      4.00000 para Root(00)
Mensaje D / 00:00:07:030 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:07:060 / Root(00) para top(01)
Mensaje * / 00:00:07:060 / top(01) para pedido(02)
Mensaje Y / 00:00:07:060 / pedido(02) / pedido_out /      5.00000 para top(01)
Mensaje D / 00:00:07:060 / pedido(02) / ... para top(01)
Mensaje Y / 00:00:07:060 / top(01) / pedido_out /      5.00000 para Root(00)
Mensaje D / 00:00:07:060 / top(01) / ... para Root(00)
