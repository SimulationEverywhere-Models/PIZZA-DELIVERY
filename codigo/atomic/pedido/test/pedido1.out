00:00:01:030 pedido_out 1
00:00:02:030 pedido_out 2
00:00:07:000 pedido_out 3
00:00:07:030 pedido_out 4
00:00:07:060 pedido_out 5
